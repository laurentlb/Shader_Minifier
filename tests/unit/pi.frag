#version 400

void main() {
    double pi = 3.141592653589793;
    double minus_pi = -3.141592653589793;
    double tau = 6.283185307179586;
    double minus_tau = -6.283185307179586;
    double half_pi = 1.5707963267948966;
    double minus_half_pi = -1.5707963267948966;
    double precise_pi = 3.14159265358979323846264338327950288419716939937510;
	gl_FragColor = vec4(pi+minus_pi+tau+minus_tau+half_pi+minus_half_pi+precise_pi);
}
