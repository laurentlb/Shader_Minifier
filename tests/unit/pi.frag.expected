#version 400

void main()
{
  double pi=acos(-1.),minus_pi=-acos(-1.),tau=2.*acos(-1.),minus_tau=-2.*acos(-1.),half_pi=acos(0.),minus_half_pi=-acos(0.),precise_pi=acos(-1.);
  gl_FragColor=vec4(pi+minus_pi+tau+minus_tau+half_pi+minus_half_pi+precise_pi);
}
